frontend/src/components/RadioButtonsTile.tsx

```diff
@@ -47,10 +47,7 @@ function RadioButtonsTile(props: Props) {
   };
 
   return (
-    <div
-      role="radiogroup"
-      className={`padding-2px ${props.isHorizontally ? "grid-row" : ""}`}
-    >
+    <div className={`padding-2px ${props.isHorizontally ? "grid-row" : ""}`}>
```

Comment:
Can we add a fieldset and a legend here to match the suggestion?
That will require to look in the code and check if there is any heading used outside of the component to move it inside.

Reply:
The `fieldset` and the `legend` elements are already present.